spec/system/admin_manage_auth_settings_spec.rb

```diff
@@ -48,4 +53,42 @@
     expect(auth_settings.enable_partial_email_signup).to be(false)
     expect(auth_settings.enable_partial_sms_signup).to be(true)
   end
+
+  it "shows a warning message when the SMS gateway service is not configured" do
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`